pkg.sh

```diff
@@ -13,6 +13,14 @@ mkdir -p bin/dexc-darwin-arm64-v${VER}
 
 export CGO_ENABLED=0
 
+# Generate the localized_html and build the webpack bundle prior to building the
+# webserver package, which embeds the files.
+pushd client/webserver/site
+go generate # should be a no-op
```

Comment:
Just wondering, what's the reason for this?

Reply:
Make sure the templates in localized_html correspond to the sources.  CI makes us check them in presently so it's likely a no-op in this script if it's being run on an unmodified git revision.